integration/raft/channel_participation_test.go

```diff
@@ -731,6 +776,7 @@ var _ = Describe("ChannelParticipation", func() {
 				Height:            7,
 			})
 		})
+
```

Comment:
Super nit that doesn't need to be addressed: I guess your editor settings like to format in this extra line?

Reply:
or me not sure 😃